packages/canary/tailwind.config.js

```diff
@@ -72,6 +89,15 @@ module.exports = {
         warning: {
           DEFAULT: 'hsl(var(--warning))'
         },
+        'success-glow': {
+          DEFAULT: 'hsl(var(--success-glow))'
+        },
+        'error-glow': {
+          DEFAULT: 'hsl(var(--error-glow))'
+        },
+        'warning-glow': {
+          DEFAULT: 'hsl(var(--warning-glow))'
+        },
```

Comment:
should not be added. 
also they need to be `hsla` to support the hardcoded alpha, or it gets ignored, defeating the purpose